builtin/rebase--interactive2.c

```diff
@@ -131,57 +139,79 @@ int cmd_rebase__interactive(int argc, const char **argv, const char *prefix)
 		warning(_("--[no-]rebase-cousins has no effect without "
 			  "--rebase-merges"));
 
-	ret = prepare_branch_to_be_rebased(&opts, switch_to);
-	if (ret)
-		return !!ret;
+	if (command == CONTINUE)
+		ret = sequencer_continue(&opts);
+	else if (command == SKIP) {
+		struct string_list merge_rr = STRING_LIST_INIT_DUP;
+
+		rerere_clear(&merge_rr);
+		ret = sequencer_continue(&opts);
+	} else if (command == EDIT_TODO)
+		ret = edit_todo_list(flags);
+	else if (command == SHOW_CURRENT_PATCH) {
+		struct child_process cmd = CHILD_PROCESS_INIT;
+
+		cmd.git_cmd = 1;
+		argv_array_pushl(&cmd.args, "show", "REBASE_HEAD", "--", NULL);
+		ret = run_command(&cmd);
+	} else {
+		const char *head_hash = NULL;
+		char *revisions = NULL, *shortrevisions = NULL;
+		struct argv_array make_script_args = ARGV_ARRAY_INIT;
+		FILE *todo_list;
+
+		ret = prepare_branch_to_be_rebased(&opts, switch_to);
+		if (ret)
+			return !!ret;
+
+		ret = get_revision_ranges(upstream, onto, &head_hash,
+					  &revisions, &shortrevisions);
+		if (ret)
+			return !!ret;
```

Comment:
When this returns, the returned value is always 1, right? In this case maybe it is clearer to have `return 1` instead of `return !!ret`.